server/channels/store/sqlstore/post_store.go

```diff
@@ -863,7 +869,7 @@ func (s *SqlPostStore) GetEtag(channelId string, allowFromCache, collapsedThread
 	if err != nil {
 		result = fmt.Sprintf("%v.%v", model.CurrentVersion, model.GetMillis())
```

Comment:
Side note: we should do this sort of logic at the upper layer and not the store layer. Because we have just skipped logging the error and we don't have any idea that an error has happened. Ideally, we log the error as warning, and then do this logic at the call site.